### PR TITLE
Feature/talk voting

### PR DESCRIPTION
**Pull request by artcz**

### CODE REVIEW

File: conference/talk_voting.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    )
+
+
+def is_user_allowed_to_vote(user):
+    """
+    Checks if user is allowed to vote at the moment of accessing this function
+    This usually means checking if they have at least one ticket associated
+    with their account (either for this or any of the past years
+    """
+    return user.ticket_set.all().exists()
```
Comment (umgelurgel) [inline conference/talk_voting.py:54]: I think last year it was only users who have attended the conference before could vote - just want to confirm this change in behaviour is intended.

**Review by umgelurgel — COMMENTED** [review]
Looks great! Just one question related to change from 2018.

File: conference/talk_voting.py (hunk 1 above)
Comment (artcz) [reply conference/talk_voting.py:54]: It's essentially "attended before or bought a ticket for this year already"

Comment (artcz) [reply conference/talk_voting.py:54]: I think last year it was the same, but if it wasn't it should have been. :)

**Pull request merged**